### ISSUE TITLE
MH Alliance base

### ISSUE THREAD

**Issue by carebot90** [Nextcalibur/Bugtracker#136]
Hi,

 

On the waves before 1st and 2d boss, if you use paladin's Crusader's Aura and maybe a mage's blink, you can land on a roof of a house near the waves entrance.

If you attack mobs from there (as a mage or warlock, for example), they won't be able to reach you.

 

On the waves before 3d and 4th boss, you can jump on a flag near the waves' entrance. The same situation.

 

I don't have the screenshots but I could show you in-game if you want. (/w Bon)

 

Yours sincerely,

Khud0.
 Soo.. we had a debate about this place with a PUG dude tonight.

He said that "A GM saw me attacking mobs from here and didn't say anything".

 

Basically, this position allows you to spam AoE from the 1st seconds of a pull, without worrying about overaggroing the tank.

Is it actually allowed? O.o

 

(Had to crop the screenshot cause of the forum limitations) 
![WoWScrnShot_022418_200948 jpg a267737f16d78c1c0811926f3f96076c](https://user-images.githubusercontent.com/63980522/79810679-e41ffa00-8373-11ea-89cb-f1cfbcae4bc2.jpg)

**Comment by carebot90** [comment]
I guess this was more of a question is it an abuse because npc can’t reach you, the bug is that maybe they should be able to reach you? Nothing else to add
